test/e2e/tests/confirmations/alerts/queued-confirmations.spec.ts

```diff
@@ -111,8 +112,18 @@ describe('Queued Confirmations', function () {
           await switchChainToDappOne(driver);
 
           await switchToDAppAndCreateTransactionRequest(driver);
+          await driver.waitUntilXWindowHandles(4);
           await switchToDAppAndCreateTransactionRequest(driver);
+          await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+          await driver.waitForSelector(
+            By.xpath("//div[normalize-space(.)='1 of 2']"),
+          );
+
           await switchToDAppAndCreateTransactionRequest(driver);
+          await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+          await driver.waitForSelector(
+            By.xpath("//div[normalize-space(.)='1 of 3']"),
+          );
```

Comment:
we make sure that after each tx is triggered, the wallet receives it, by rendering the navigation